Modules/Sources/WordPressUI/Views/Users/Views/UserDetailsView.swift

```diff
@@ -0,0 +1,160 @@
+import SwiftUI
+
+struct UserDetailsView: View {
+
+    private let userProvider: UserDataProvider
+    private let actionDispatcher: UserManagementActionDispatcher
+    let user: DisplayUser
+
+    @StateObject
+    var viewModel: UserDetailViewModel
+
+    @Environment(\.dismiss)
+    var dismissAction: DismissAction
+
+    init(user: DisplayUser, userProvider: UserDataProvider, actionDispatcher: UserManagementActionDispatcher) {
+        self.user = user
+        self.userProvider = userProvider
+        self.actionDispatcher = actionDispatcher
+        _viewModel = StateObject(wrappedValue: UserDetailViewModel(userProvider: userProvider))
+    }
+
+    var body: some View {
+        Form {
+            VStack {
+                UserProfileImage(size: 96, url: user.profilePhotoUrl)
+                Text(user.displayName)
+                    .font(.title)
+                Text(user.handle)
+                    .font(.title2)
+                    .foregroundStyle(.secondary)
+            }
+            .frame(maxWidth: .infinity)
+            .listRowBackground(Color.clear)
+            .listRowInsets(.zero)
+
+            Section {
+                makeRow(title: Strings.roleFieldTitle, content: user.role)
+                makeRow(title: Strings.emailAddressFieldTitle, content: user.emailAddress, link: user.emailAddress.asEmail())
+                if let website = user.websiteUrl, !website.isEmpty {
+                    makeRow(title: Strings.websiteFieldTitle, content: website, link: URL(string: website))
+                }
+                if let biography = user.biography, !biography.isEmpty {
+                    makeRow(title: Strings.bioFieldTitle, content: biography)
+                }
+            }
+
+            if viewModel.currentUserCanModifyUsers {
+                Section(Strings.accountManagementSectionTitle) {
+                    NavigationLink {
```

Comment:
Let's show these as modal screens.

Reply:
In follow-up PRs, I'll use an alert for the "set new password" feature and a modal/sheet for deleting users.